metrics/scaling/k8s_scale_nc.sh

```diff
@@ -437,8 +437,8 @@ help()
 	usage=$(cat << EOF
 Usage: $0 [-h] [options]
    Description:
-	Launch a series of workloads and take memory metric measurements after
-	each launch.
+	Launch a series of workloads and take pod query-response latency measurements
```

Comment:
IMHO `query-response` can be redundant, latency in most of the cases refers to the time that something takes to response.

Reply:
I've taken this description from the ongoing [PR212](https://github.com/clearlinux/cloud-native-setup/pull/212/files#diff-49c99b2e785db0c2676430f5704887deR54) of such test. 

@askervin any thoughts on this?

Reply:
I think `query-response` is fine (although you may wish to add 'http' or whatever protocol is being used to it). Without saying `query-response`, then I don't think we'd know which latency was being measured (network, boot, etc.).